lib/flyover_comments/authorization.rb

```diff
@@ -21,5 +21,15 @@ def can_create_flyover_comment?(comment, user)
       end
     end
 
+    def can_flag_flyover_comment?(comment, user)
+      if Object.const_defined?("Pundit") && policy = Pundit.policy(user, comment)
+        policy.create?
+      elsif user.respond_to?(:can_flag_flyover_comment?)
+        user.can_flag_flyover_comment?(comment)
+      else
```

Comment:
Let's return `!user.nil?` from the else clause. I think `comment.user == user` was a sensible default for creating a comment, but only being able to flag your own comments probably doesn't make sense as the default behavior. For flags, I think a signed in user is probably a sensible default.